packages/babel-generator/src/generators/flow.js

```diff
@@ -331,6 +331,12 @@ export function ObjectTypeProperty(node: Object) {
   this.print(node.value, node);
 }
 
+export function ObjectTypeSpreadProperty(node: Object) {
+  this._variance(node);
```

Comment:
Since this node type can't have a variance annotation, this line can be removed.